### DISCUSSION TITLE
Register-PnPEntraIDApp : The term 'Register-PnPEntraIDApp' is not recognized as the name of a cmdlet, function, script  file, or operable program.

### DISCUSSION THREAD

**Discussion by packercountryusa** [pnp/powershell#4273] [Q&A]
I am trying to follow the video instructions to Install & register the PnP PowerShell application, however receive the dreaded 'not recognized' error message.

I have tried to research this, but am unsure what is required to make this function...

I currently have PnP.PowerShell module 2.12.0 installed, using Windows PowerShell ISE...

**Comment by gautamdsheth** [comment]
Hi @packercountryusa , this requires you to use PowerShell 7.2 or later versions of PowerShell .
